roq-deployment/amazon/GCM/tasks/main.yml

```diff
@@ -0,0 +1,11 @@
+- name: Creates .roq directory
+  file: path=~/.roq state=directory mode=777
```

Comment:
The ~/.roq folder could probably be a variable, since you use it in several places.